_partials/keep_current.md

```diff
@@ -0,0 +1,307 @@
+
+# Keeping your setup current
+
+This section contains the steps that you need to follow in order to make sure that your setup is up to date
+
+First and foremost, in order to work in good conditions, make sure that :
+- you are working with a high speed internet connection
+- your computer has enough memory (8GB), so that you can run your code efficiently
+- your computer has enough disk space (30GB), so that you can work on big datasets
+
+## git
+
+Verify that git works
+
+``` bash
+git --version
+```
+
+👉 You should expect an output similar to this one yielding the git version :
+
+``` bash
+git version 2.33.0
+```
+
+## GitHub
+
+Verify that GitHub works and that you have access to the Le Wagon repositories
+
+``` bash
+cd ~/code/<YOUR_GITHUB_NICKNAME>/
+git clone git@github.com:lewagon/data-certification-exam data-certification-exam-test-delete-me
+```
+
+👉 The repo should clone correctly :
+
+``` bash
+Cloning into 'data-certification-exam-test-delete-me'...
+remote: Enumerating objects: 21, done.
+remote: Counting objects: 100% (21/21), done.
+remote: Compressing objects: 100% (14/14), done.
+Receiving objects: 100% (21/21), done.
+Resolving deltas: 100% (6/6), done.
+remote: Total 21 (delta 6), reused 16 (delta 1), pack-reused 0
+```
+
+👉 You can delete the cloned repo
+
+``` bash
+rm -Rf data-certification-exam-test-delete-me
+```
+
+## Verify your pyenv configuration
+
+Verify that you have a `~/.zprofile` :
+
+``` bash
+cat ~/.zprofile
+```
+
+👉 You should see the following lines :
+
+``` bash
+# Setup the PATH for pyenv binaries and shims
+export PYENV_ROOT="$HOME/.pyenv"
+export PATH="$PYENV_ROOT/bin:$PATH"
+type -a pyenv > /dev/null && eval "$(pyenv init --path)"
+```
+
+👉 If the command does not output anything, create the `~/.zprofile` file :
+
+``` bash
+cd
+touch .zprofile
+```
+
+Add the following lines to your `~/.zprofile` :
+
+``` bash
+# Setup the PATH for pyenv binaries and shims
+export PYENV_ROOT="$HOME/.pyenv"
+export PATH="$PYENV_ROOT/bin:$PATH"
+type -a pyenv > /dev/null && eval "$(pyenv init --path)"
+```
+
+## Create a dedicated virtual environment
+
+Update pyenv :
+
+$MAC_START
+``` bash
+brew update && brew upgrade pyenv
+```
+$MAC_END
+$LINUX_START
+``` bash
+cd $(pyenv root) && git pull
+```
+$LINUX_END
+$WINDOWS_START
+``` bash
+cd $(pyenv root) && git pull
+```
+$WINDOWS_END
+
+Install the current python version :
+
+```bash
+pyenv install <PYTHON_VERSION>
+```
+
+👉 Make sure that the command completes correctly
+
+Let's make sure no virtual environment with the same name exists :
+
+```bash
+pyenv virtualenv-delete lewagon_current
+```
+
+Create a new virtual environment :
+
+```bash
+pyenv virtualenv <PYTHON_VERSION> lewagon_current
+```
+
+Set the new virtual environment as default :
+
+```bash
+pyenv global lewagon_current
+```
+
+You should now be able to see the new virtual environment as active :
+
+``` bash
+pyenv versions
+```
+
+👉 Here is the expected output :
```

Comment:
I think _Expected_ is too strong, there are a lot of possibilities as an output there